apps/tx-builder/src/lib/checksum.test.js

```diff
@@ -0,0 +1,103 @@
+import { addChecksum, validateChecksum } from './checksum';
```

Comment:
Why are tests written in javascript and not typescript?

Reply:
How is the `lib` folder different from the `services` folder? I'd avoid spreading logic code across different folders

Reply:
I use this test file for helping me to  develop the checksum feature and left it in the code. Is the only test file in the project right now and didn't see the benefit of converting it to TS for this concrete case to be honest

Reply:
going to move all to the lib folder!